google-cloud-video-stitcher/samples/create_live_config.rb

```diff
@@ -0,0 +1,59 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START videostitcher_create_live_config]
+require "google/cloud/video/stitcher"
+
+##
+# Create a live config
+#
+# @param project_id [String] Your Google Cloud project (e.g. "my-project")
+# @param location [String] The location (e.g. "us-central1")
+# @param live_config_id [String] Your live config name (e.g. "my-live-config")
+# @param source_uri [String] Uri of the live stream to stitch
+#                            (e.g. "https://storage.googleapis.com/my-bucket/main.mpd")
+# @param ad_tag_uri [String] Uri of the ad tag (e.g. "https://pubads.g.doubleclick.net/gampad/ads...")
+# @param slate_id [String] The default slate ID to use when no slates are specified in an ad break's message
```

Comment:
Some of these lines are quite long.

Reply:
applied 80-column limit (where possible)